server/datastore/mysql/queries.go

```diff
@@ -4,10 +4,15 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
-
 	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
 	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/go-kit/log/level"
 	"github.com/jmoiron/sqlx"
+	"strings"
+)
+
+const (
+	statsLiveQueryType = 1
 )
```

Comment:
this may be clearer as to the difference between live and scheduled as I'm not sure `0` is defined anywhere

```suggestion
const (
	statsScheduledQueryType = iota
	statsLiveQueryType
)
```

Reply:
on second thought, since `statsScheduledQueryType` isn't used, does it make more sense to use `statsLiveQueryType` as a boolean?  ie.  `isStatsLiveQueryType`?

Reply:
I don't think it matters that much. Non-boolean is more scalable in case we come up with new query types later.